lobsterpy/cohp/analyze.py

```diff
@@ -43,29 +42,27 @@ class Analysis:
         are_cobis : bool indicating if file contains COBI/ICOBI data
         are_coops : bool indicating if file contains COOP/ICOOP data
         noise_cutoff : float that sets the lower limit of icohps or icoops or icobis considered
-        set_cohps: list of cohps
-        set_coordination_ions: list of coodination environment strings for each cation
+        seq_cohps: list of cohps
+        seq_coord_ions: list of co-ordination environment strings for each cation
         set_equivalent_sites: set of inequivalent sites
         seq_ineq_ions: set of inequivalent cations/sites in the structure
         seq_infos_bonds (list): information on cation anion bonds (lists

```

Comment:
Shouldn't you then rename all of them if you want to indicate sequence instead of set?

Reply:
Ops, I just changed for the attributes which were changed in the docstring. Did not see that now there was inconsistency. Will rename them 🙈

Reply:
Done